variables.md

```diff
@@ -482,3 +482,14 @@ Variable        | Type      | Default value | Details
 ---             | ---       | ---           | ---
 SCC_RECGODE_LTSS | string | | This will hold the registration code for activating the product SLES-LTSS
 SCC_RECGODE_LTSS_ES | string | | This will hold the registration code for activating the product SLES-LTSS-Extended-Security
+
+### Agama specific variables
+
+Following variables are relevant for agama installation
+
+Variable        | Type      | Default value | Details
+---             | ---       | ---           | ---
+AGAMA | boolean | 0 | Agama installation support
+AGAMA_AUTO | string | | The auto-installation is started by passing `agama.auto=<url>` on the kernel's command line
+AGAMA_LIVE_ISO | string | | The url of agama installer iso image which defined in linuxrc command line. You can use the format like "root=live:http://agama.iso"
```

Comment:
AGAMA_LIVE_ISO_URL | string | | The url of agama live iso to pass as kernel's command-line parameter. Example of use "root=live:http://agama.iso".